app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -320,6 +327,8 @@
     <string name="settings_eh_thumb_resolution">縮圖解析度</string>
     <string name="settings_eh_thumb_resolution_summary">%s，指定解析度可能會使縮圖取得失敗</string>
     <string name="settings_eh_thumb_resolution_auto">自動</string>
+    <string name="settings_eh_force_eh_thumb">強制 e-hentai 縮圖/API 伺服器</string>
+    <string name="settings_eh_force_eh_thumb_summary">若exhentai載入遇到問題可嘗試此項</string>
```

Comment:
I just updated this string, you may want to check it https://github.com/FooIbar/EhViewer/commit/256dedcb7cec886fe9c4bddcda0098f5f8e37a13